build.yaml

```diff
@@ -43,6 +43,18 @@ schedules:
         - python: [3.4, 3.6]
         - cassandra: ['2.0', '2.1', '3.0']
 
+  commit_branches_dev:
+    schedule: per_commit
+    branches:
+      include: [/dev.*/]
+    env_vars: |
+      EVENT_LOOP_MANAGER='libev'
+      EXCLUDE_LONG=1
+    matrix:
+      exclude:
+        - python: [2.7, 3.4, 3.6]
+        - cassandra: ['2.0', '2.1', '2.2', '3.0', 'test-dse']
+
```

Comment:
dev purpose... when I code.. I only one to see results for the latest versions.. not a whole matrix. will be rename to dev-python-*